src/stores/AppStore.js

```diff
@@ -17,36 +15,22 @@ export default class AppStore {
 
   @action async fetchApps() {
     this.isLoading = true;
-    const result = await request.get('api/apps');
+    const result = await request.get('api/v1/apps');
```

Comment:
the api version should be a global env variable, when request function initialize need check this `api version number` to construct rest api params automatically

Reply:
this will be discussed in the future

Reply:
ok